app/buyers/views/buyers.py

```diff
@@ -127,6 +127,10 @@ def view_brief_overview(framework_slug, lot_slug, brief_id):
     sections = content.summary(brief)
     delete_requested = True if request.args.get('delete_requested') else False
 
+    content_loader.load_messages(brief['frameworkSlug'], ['urls'])
+    call_off_contract_url = content_loader.get_message(brief['frameworkSlug'], 'urls', 'call_off_contract_url')
```

Comment:
Other places (supplier framework agreement page, framework dashboard, the homepage) don't do any error handling. They just set a variable and pass it to the template.